packages/fxa-auth-server/lib/payments/stripe.js

```diff
@@ -217,6 +219,17 @@ class StripeHelper {
     );
   }
 
+  /**
+   * Fetch a list of subscriptions for a customer from Stripe.
+   *
+   * @param {string} customerId
+   * @param {SubscriptionListParams} options
+   * @returns {Promise<Subscriptions>}
+   */
+  async fetchAllSubscriptionsForCustomer(customerId, options = {}) {
```

Comment:
Do we need this method? We have access to the cached customer record in [`customer()`](https://github.com/mozilla/fxa/blob/9589fc7906810c085ce755852f42804476dca8e5/packages/fxa-auth-server/lib/payments/stripe.js#L193) and non-cached fetch via [`fetchCustomer()`](https://github.com/mozilla/fxa/blob/9589fc7906810c085ce755852f42804476dca8e5/packages/fxa-auth-server/lib/payments/stripe.js#L165) - both of these [include subscriptions](https://github.com/mozilla/fxa/blob/9589fc7906810c085ce755852f42804476dca8e5/packages/fxa-auth-server/lib/payments/stripe.js#L215). I'd say ideally use the cached customer data, but you can use the direct fetch if the latest data is absolutely necessary. 

(I might also suggest that if stale data is a concern during support requests, we should call `deleteCachedCustomer()` somewhere in that process to refresh the cache)

Reply:
This is called _only_ when the subscriptions list in customer has the `has_more` set to `true`.  I'm handling that for completeness sake, but practically no customer will have that many subscriptions any time soon.  (I hope I'm very wrong on that though. 😁 )

Reply:
> This is called only when the subscriptions list in customer has the has_more set to true. 

Hmm, if we're missing subscriptions in `fetchCustomer`, we need to fix that. That call *should* include *all* subscriptions because the code that determines enabled capabilities counts on that. (i.e. if any subscriptions are missing, RPs will not get the data in the profile that enables the service)

If *that's* the reason for this additional method, it sounds like a bug that needs to be fixed instead.

Reply:
> Hmm, if we're missing subscriptions in `fetchCustomer`, we need to fix that. That call _should_ include _all_ subscriptions...

Ok, let's make that change then.

Reply:
Augmented `fetchCustomer`: https://github.com/mozilla/fxa/pull/3963/files#diff-26644f246b5f22f6205561269d539ce7R182-R190